compiler/damlc/daml-stdlib-src/DA/Date.daml

```diff
@@ -38,6 +38,14 @@ deriving instance Show Month
 deriving instance Enum Month
 deriving instance Bounded Month
 
+instance Enum Date where
+  toEnum = daysSinceEpochToDate
+  fromEnum = dateToDaysSinceEpoch
+  
```

Comment:
```suggestion
  enumFrom     x   = enumFromTo     x maxBound
  enumFromThen x y = enumFromThenTo x y bound
     where
         bound | fromEnum y >= fromEnum x = maxBound
                    | otherwise                = minBound
  
```